src/contracts/interfaces/IPaymentCoordinator.sol

```diff
@@ -0,0 +1,173 @@
+// SPDX-License-Identifier: BUSL-1.1
+pragma solidity =0.8.12;
+
+import "@openzeppelin/contracts/token/ERC20/IERC20.sol";
+import "./IStrategy.sol";
+
+/**
+ * @title Interface for the `IPaymentCoordinator` contract.
+ * @author Layr Labs, Inc.
+ * @notice Terms of Service: https://docs.eigenlayer.xyz/overview/terms-of-service
+ */
+interface IPaymentCoordinator {
+
+    /// STRUCTS ///
+    struct StrategyAndMultiplier {
+        IStrategy strategy;
+        uint96 multiplier;
+    }
+	struct RangePayment {
+        // Strategies & relative weights of shares in the strategies
+        StrategyAndMultiplier strategiesAndMultlipliers;
+        IERC20 token;
+        uint256 amount;
+        uint256 startTimestamp;
+        // duration in seconds
+        uint256 duration;
+	}
+
+    struct DistributionRoot {
+        // timestamp after which the root can be claimed against
+        uint32 activatedAfter;
+        // merkle root of the distribution
+        bytes32 root;
+    }
+
+    struct ClaimsTreeMerkleLeaf {
+        IERC20 token; 
+        uint256 amount;
+        // Explicit recipient of the claim
+        address recipient;
+    }
+
+    struct PaymentMerkleClaim {
+        ClaimsTreeMerkleLeaf[] leaves;
+        // The index of the root in the list of roots
+        uint32 rootIndex;
+        // proof of the claimaint's account root in the Merkle tree
+        bytes accountTreeProof;
+        // The indices and proofs of the leafs in the claimaint's merkle tree for this root
+        uint32[] leafIndices;
+        bytes[] tokenTreeProofs;
+    }
+
+    /// EVENTS ///
+
+    event RangePaymentCreated(address indexed avs, RangePayment rangePayment);
+    event PaymentUpdaterSet(address indexed oldPaymentUpdater, address indexed newPaymentUpdater);
```

Comment:
Probably worth logging the governor address for this too.

Reply:
like, the current owner?
guess I am assuming this contract inherits from OZ's Ownable or similar